examples/04_07_specific_words_comparison_el_vestnik.ipynb

```diff
@@ -0,0 +1,591 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Primerjava pristopov za luščenje ključnih besed na anotiranih besedilih iz revije Elektrotehniški vestnik\n",
+    "\n",
+    "V tem zvezku predstavljamo primerjavo pristopov za luščenje ključnih besed iz besedil iz revije Elektrotehniški vestnik, ki imajo označene ključne besede. Ključne besede so označene s strani avtorjev člankov.\n",
+    "\n",
+    "Predstavili bomo objektivno primerjavo pristopov za luščenje besed. Primerjali bomo izluščene besede z ključnimi besedami, ki so jih označili avtorji člankov. Za vsako metodo bomo prikazali povprečno preciznost (precision), priklic (recall) in mero F1."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import string\n",
+    "import ast\n",
+    "\n",
+    "import nltk\n",
+    "from textsemantics.server_api import ServerAPI\n",
+    "from nltk.tokenize import RegexpTokenizer\n",
+    "from nltk.corpus import stopwords\n",
+    "from lemmagen.lemmatizer import Lemmatizer\n",
+    "from lemmagen import DICTIONARY_SLOVENE\n",
+    "from matplotlib import pyplot as plt\n",
+    "import yake\n",
+    "from gensim.summarization import keywords as gensim_kw\n",
+    "import rake\n",
+    "\n",
+    "from utils import specific_words"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Naložimo metapodatke in dokumente iz zbirke _elektrotehniski-vestnik-clanki_ - članki objavljeni v reviji Contributions to Contemporary History. Dokumente preprocesiramo in izluščimo posamezne besede."
```

Comment:
v reviji Contributions to Contemporary History --> v reviji Elektrotehniški vestnik
copy-paste 🙈

Reply:
Uf pa sem mislil, da sem vse zamenjal.